CMakeLists.txt

```diff
@@ -413,39 +412,29 @@ mark_as_advanced(CUDA_NVCC_FLAGS_DEVICEDEBUG)
 # we might use:
 # set_property(TARGET tgt PROPERTY CXX_STANDARD 11) in the future but need to check support in older cmake version
 
-set(CMAKE_CXX_FLAGS_DEVEL  "${OpenMP_CXX_FLAGS} -std=c++11 -O3 -Wall" CACHE STRING
-    "Flags used by the CUDA compiler during regular development builds."
-    FORCE )
+set(CMAKE_CXX_FLAGS_DEVEL  "${OpenMP_CXX_FLAGS} -std=c++11 -Ofast -Wall" CACHE STRING
```

Comment:
Fixed (reverted back to O3).  It was an accidental inclusion, best reverted since I don't know if Ofast is supported by all compilers (e.g., clang).